tools/test/test_depsolve.py

```diff
@@ -251,26 +251,23 @@ def cache_dir_fixture(tmpdir_factory):
     return str(tmpdir_factory.mktemp("cache"))
 
 
-@pytest.mark.parametrize("test_case", test_cases)
-def test_depsolve(repo_servers, test_case, cache_dir):
+def _test_depsolve(command, repo_servers, test_case, cache_dir):
```

Comment:
LGTM, one question here - if the `command` should not be the last arg? In the `depsolve()` it is last arg so maybe we should stick to this order?

Reply:
It's a really good question - my thinking was that it's the "most important" thing that is passed to _test_depsolve() so it deserves to be first - but I understand if people have different opinions about this of course :)